pty_openbsd.go

```diff
@@ -29,8 +40,8 @@ func open() (pty, tty *os.File, err error) {
 		return nil, nil, err
 	}
 
-	pty = os.NewFile(uintptr(ptm.Cfd), "/dev/ptm")
-	tty = os.NewFile(uintptr(ptm.Sfd), "/dev/ptm")
+	pty = os.NewFile(uintptr(ptm.Cfd), cInt8ToString(ptm.Cn[:]))
```

Comment:
Isn't this going to cause a memory leak? https://pkg.go.dev/cmd/cgo mention it is the caller responsibility to free memory. Not familiar with .Cn / .Sn, so I may be off.

Reply:
These come from the ptmget struct defined in:

  https://github.com/creack/pty/blob/master/ztypes_openbsd_32bit_int.go#L7

Each of these are a fixed size array [16]int8 - the ptmget is Go allocated at:

  https://github.com/creack/pty/blob/master/pty_openbsd.go#L27

Before being passed to the `ioctl(2)` call - this is copying the data out of this existing allocation.